smk-config.json

```diff
@@ -101,7 +101,7 @@
+            "popupTemplate": "<div class=\"smk-body\">\n    <div class=\"smk-feature\">\n        <div class=\"smk-layer\">\n            Municipal Regional District Tax: 2%\n        </div>\n        <div class=\"smk-attributes\">\n            <div class=\"smk-attribute\">\n                <span class=\"smk-attribute-title\">\n                    Tax Rate:\n                </span>\n                <span>\n                    <span class=\"smk-value\">\n                        {{feature.properties.Tax_Rate}}\n                    </span>\n                </span>\n            </div>\n            <div class=\"smk-attribute\">\n                <span class=\"smk-attribute-title\">\n                    Accommodation Area:\n                </span>\n                <span>\n                    <span class=\"smk-value\">\n                        {{feature.properties.Accom_Area}}\n                    </span>\n                </span>\n            </div>\n            <div class=\"smk-attribute\">\n                <span class=\"smk-attribute-title\">\n                    MRDT Start date:\n                </span>\n                <span>\n                    <span class=\"smk-value\">\n                        {{(new Date(feature.properties.MRDT_Start_Date)).toLocaleDateString()}}\n                    </span>\n                </span>\n            </div>\n            <div class=\"smk-attribute\">\n                <span class=\"smk-attribute-title\">\n                    Current MRDT Rate Effective Date:\n                </span>\n                <span>\n                    <span class=\"smk-value\">\n                        {{(new Date(feature.properties.Tax_Rate_Eff_Date)).toLocaleDateString()}}\n                    </span>\n                </span>\n            </div>\n            <div class=\"smk-attribute\">\n                <span class=\"smk-attribute-title\">\n                    TACS Jurisdiction:\n                </span>\n                <span>\n                    <span class=\"smk-value\">\n                        {{feature.properties.TACS_Juris}}\n                    </span>\n                </span>\n            </div>\n        </div>\n    </div>\n</div>"
```

Comment:
This looks good, and I confirmed indenting is preserved in smk-cli when viewing the template.

I thought we might need to use Date.UTC() but the value is epoch time so that's not necessary.